cc/mcts_node.h

```diff
@@ -126,7 +126,7 @@ class MctsNode {
   std::array<EdgeStats, kNumMoves> edges;
 
   // TODO(tommadams): a more compact representation.
```

Comment:
this would fix that 'compactness' TODO, i think ;)

Reply:
This would still be 3Gigs of memory overhead at 1M playouts.
it can be compacted by a factor of 8 with clever boolean logic (or vector<bool>) or removed entirely by integrating it into U

Reply:
Isn't it more like 362MB?
I don't think moving to bit twiddling is really necessary at this point but if it were, we should be able to just drop in std::bitset.